web/toggle_button.css

```diff
@@ -44,31 +46,35 @@
     --border-interactive-color-hover: SelectedItem;
     --border-interactive-color-active: SelectedItem;
     --border-interactive-color-disabled: GrayText;
+    --border-color-interactive: ButtonText;
+    --border-color-interactive-hover: SelectedItem;
+    --border-color-interactive-active: ButtonText;
     --color-canvas: ButtonText;
+    --background-color-canvas: Canvas;
   }
+}
 
-  /* 
+/*
```

Comment:
This change looks unrelated, let's revert it.

Reply:
It has been added by the linter... so I think we can keep it.